fhd_core/visibility_manipulation/vis_param_extract.pro

```diff
@@ -4,7 +4,12 @@ FUNCTION vis_param_extract,params,header, antenna_mod_index=antenna_mod_index
 uu_arr=Double(reform(params[header.uu_i,*]))
 vv_arr=Double(reform(params[header.vv_i,*]))
 ww_arr=Double(reform(params[header.ww_i,*]))
-time=reform(params[header.date_i,*])
+
+; Use both date fields if available in the uvfits file
+; The center value is not added automatically, so add jd0
```

Comment:
You added the comment about adding `jd0`, but then removed `jd0` in a later commit. It looks like the problem with the beam if `jd0` is included is that the double precision float overflows. That is probably why we avoided adding it in in the first place, so maybe all you need to do is update this comment.

Reply:
Looks like julian dates are actually calculated in `fhd_struct_init_meta`. I have updated the comment to reflect this.